homework/81333/golang-highlighter/src/Data/TokenTypes.hs

```diff
@@ -0,0 +1,289 @@
+{-# LANGUAGE GADTs             #-}
```

Comment:
Why do you need `GADTs` here, a normal data type does the same job, as far as I can see?